README.md

```diff
@@ -194,3 +195,7 @@ Desktop Tool | Description
 [Flutter](https://flutter.dev/) | Flutter is Google’s UI toolkit for building beautiful, natively compiled applications for mobile, web, and desktop from a single codebase. It is free and open source. 
 [Pexels](https://www.pexels.com) | Pexels. Free stock images, photos and videos.
 [WSL2](https://docs.microsoft.com/en-us/windows/wsl/) | Windows Subsystem for Linux allows you to run a Linux shell and run Linux commands on Windows 10.
+[GitHub CLI](https://cli.github.com/) | GitHub CLI brings GitHub to your terminal. Free and open source.
+[Icon8](https://icons8.com/) | Icons8 allows designers to express their ideas giving the right design elements at the right time.
```

Comment:
is it a desktop tool?

Reply:
GitHub CLi running on terminal, Icons8 is desktop tools

Reply:
i did not find a way to download the Icons8, can you send me a download link?

Reply:
There is a desktop version https://icons8.com/app available for mac, windows, figma, photoshop

Reply:
Nice! Please, change the URL and i'll merge it :)